### PR TITLE
Add support for unique ID in database request / responses

### PR DESCRIPTION
**Pull request by rhart92**
Now `createDatabase` and `updateDatabase` allow creating unique ID properties and `getDatabase` will now return unique ID properties.

### CODE REVIEW

File: src/api-endpoints.ts [hunk 1]
```diff
@@ -5210,6 +5218,7 @@ type LanguageRequest =
   | "mathematica"
   | "mermaid"
   | "nix"
+  | "notion formula"
```
Comment (parulbaweja) [inline src/api-endpoints.ts:5221]: why was this added?

**Review by parulbaweja — APPROVED** [review]

File: src/api-endpoints.ts (hunk 1 above)
Comment (rhart92) [reply src/api-endpoints.ts:5221]: Just an artifact of someone adding a new language but not publishing the `sdk` 😢
